ui/js/models/resultsets_store.js

```diff
@@ -255,14 +255,12 @@ treeherder.factory('ThResultSetStore', [
         };
 
         var getSelectedJob = function(repoName){
-            return { el:repositories[repoName].lastJobElSelected,
```

Comment:
If you're going to take the approach above (of getting it via a selector after updating the job tables), you should revert this change.